share/examples/bitcoin.conf

```diff
@@ -156,6 +156,10 @@
 # Maintain coinstats index used by the gettxoutsetinfo RPC (default: 0).
 #coinstatsindex=1
 
+# Maintains an index of all transactions that have ever happened, which you can query using the remote procedure 
```

Comment:
- If you are still maintaining this proposal, you would need to remove the trailing space here for the CI linter task to pass. You can verify your change locally by running `test/lint/lint-whitespace.py`.

- Maybe can be shortened, similar to the existing coinstats entry:
```diff
-# Maintains an index of all transactions that have ever happened, which you can query using the remote procedure 
-# call (RPC) method `getrawtransaction` or the RESTful API call `get-tx`.
+# Maintain transaction index used by the getrawtransaction RPC (default: 0).
 #txindex=1
```